src/main/java/org/folio/inventory/dataimport/handlers/matching/preloaders/OrdersPreloaderHelper.java

```diff
@@ -19,6 +20,9 @@
 
 public class OrdersPreloaderHelper {
 
+    private static final String ORDER_LINES_CQL_PATTERN = "purchaseOrder.workflowStatus==Open AND %s";
+    private static final String VRN_VALUE_CQL_PATTERN = "\"\\\"refNumber\\\":\\\"%s\\\"\"";
```

Comment:
In documentation is specified another way to query this type of values: https://github.com/folio-org/raml-module-builder/blob/master/README.md#cql-matching-array-elements

`property =/@type1 value`

order-storage has the same structure in schema.json 
![image](https://user-images.githubusercontent.com/5904625/163780122-b8eecc76-3e84-4f1a-9be4-a54c23d9ab75.png)


please test this type of query